app/domain/entity/Category.java

```diff
@@ -2,13 +2,22 @@
 
 import  java.util.Date;
 
+import static java.util.UUID.randomUUID;
+
 public class Category {
 
     private String id;
     private String name;
     private String type;
     private Date CreatedDate;
 
+    public Category(String name, String type){
+        this.id = randomUUID().toString();
```

Comment:
@Bukarachka there is no point in code duplications here. You can reuse another constructor by passing parameters. Use this(args) to call another constructor